### PR TITLE
Feature/DTGB-432: 403 page

### PR DESCRIPTION
**Pull request by meirege**
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

**Comment by delrueba** [comment]
Screenshots please

File: styleguide/components/21-atoms/inputs/input-submit/input-submit.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <input type="submit"
        value="{{ value }}"
-       class="btn {% if modifier %}{{ modifier }}{% endif %}"
+       class="{% if modifier %}{{ modifier }}{% endif %}"
```
Comment (delrueba) [inline styleguide/components/21-atoms/inputs/input-submit/input-submit.twig:3]: :+1:

**Comment by delrueba** [comment]
Links do not have sufficient contrast.

File: styleguide/components/31-molecules/search/_search.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.search {
+  display: flex;
+  position: relative;
+  flex-wrap: wrap;
+
+  label {
+    @extend %h5;
```
Comment (delrueba) [inline styleguide/components/31-molecules/search/_search.scss:7]: This label does not provide sufficient information:
'are you looking for another page?' ... I would expect a checkbox.

Better not corrupt the label to look like a h5 and just use a heading element and use aria-label on the search field.
(we'll have to do this in the header too anyway).

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/31-molecules/search/_search.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/search/_search.scss:7]: I changed it to a title and fixed aria-label of the input-search atom.

**Review by delrueba — APPROVED** [review]

**Pull request merged**